### ISSUE TITLE
Set up the website and publish a release

### ISSUE THREAD

**Issue by alex-konovalov** [vlaver/Thelma#2]
At the moment, the package has no webpage, and there is no concept of releases - a user can only clone the repository or download the zip archive with the latest source code (even without the pre-built documentation). This is suitable for trying to use the package, but eventually should be addressed properly.

For packages hosted on GutHub, we offer some tools which relieve technical burden of setting up a webpage, publishing releases, and remembering all steps in the right order. First, they automate the procedure; Second, someone who has necessary permissions may be able to publish a release for you. The tools are
* https://github.com/gap-system/ReleaseTools 
* https://github.com/gap-system/GitHubPagesForGAP

and their documentation explains how to set them up. I will be happy to help to set this up.

Note that trying to load the package in the GAP master branch triggers warnings:
```
gap> LoadPackage("thelma");
#E  component `ArchiveURL' must be bound to a string started with http://, https:// or ftp://
#E  component `ArchiveURLSubset' must be bound to a list of strings denoting relative paths to readable files or directories
#E Validation of package thelma from /Users/alexk/Library/Preferences/GAP/pkg/Thelma failed
----------------------------------------------------------------
Loading  Thelma 1.00, a package for threshold logic
For help, type: ?Thelma: 
----------------------------------------------------------------
true
```
and the tools from above will help to set proper values for these fields.

**Comment by vlaver** [comment]
> At the moment, the package has no webpage, and there is no concept of releases - a user can only clone the repository or download the zip archive with the latest source code (even without the pre-built documentation). This is suitable for trying to use the package, but eventually should be addressed properly.
> 
> For packages hosted on GutHub, we offer some tools which relieve technical burden of setting up a webpage, publishing releases, and remembering all steps in the right order. First, they automate the procedure; Second, someone who has necessary permissions may be able to publish a release for you. The tools are
> 
> * https://github.com/gap-system/ReleaseTools
> * https://github.com/gap-system/GitHubPagesForGAP
> 
> and their documentation explains how to set them up. I will be happy to help to set this up.
> 
> Note that trying to load the package in the GAP master branch triggers warnings:
> 
> ```
> gap> LoadPackage("thelma");
> #E  component `ArchiveURL' must be bound to a string started with http://, https:// or ftp://
> #E  component `ArchiveURLSubset' must be bound to a list of strings denoting relative paths to readable files or directories
> #E Validation of package thelma from /Users/alexk/Library/Preferences/GAP/pkg/Thelma failed
> ----------------------------------------------------------------
> Loading  Thelma 1.00, a package for threshold logic
> For help, type: ?Thelma: 
> ----------------------------------------------------------------
> true
> ```
> and the tools from above will help to set proper values for these fields.

Dear Alex! I made the web page as you suggested: https://vlaver.github.io/Thelma/
In this release I also tried to resolve all other issues that you have mentioned. Thanks for your remarks and suggestions, they were very useful!

**Comment by alex-konovalov** [comment]
Nice, the page looks good - thanks @vlaver! I will look at the rest of changes later.

**Comment by alex-konovalov** [comment]
@vlaver Now redistributed with GAP: included in GAP 4.10.1 release, see https://www.gap-system.org/Packages/thelma.html
